.github/workflows/clp-core-build.yaml

```diff
@@ -5,49 +5,31 @@ on:
     paths:
       - ".github/actions/**"
       - ".github/workflows/clp-core-build.yaml"
+      - ".github/workflows/clp-core-build-containers.yaml"
```

Comment:
Where is this file?

Reply:
Good catch. Not enough brain power at the time of writing I guess.